apis/apps/v1alpha1/cloneset_types.go

```diff
@@ -82,6 +82,12 @@ type CloneSetSpec struct {
 
 	// Lifecycle defines the lifecycle hooks for Pods pre-available(pre-normal), pre-delete, in-place update.
 	Lifecycle *appspub.Lifecycle `json:"lifecycle,omitempty"`
+
+	// ProgressDeadlineSeconds specifies the maximum time for the CloneSet to reach available
+	// or paused state due to partitioning. The controller will set a ProgressDeadlineExceeded
+	// condition when timeout occurs, while excluding paused state duration from the deadline calculation.
+	// It continues processing failed CloneSets and defaults to 600 seconds which can be overridden explicitly.
```

Comment:
plz correct the default description, and the docs in the openkruise.io should be also updated